composer.json

```diff
@@ -37,7 +37,11 @@
         "spryker/propel": "*",
         "spryker/router": "*",
         "spryker/silex": "*",
-        "spryker/testify": "*"
+        "spryker/testify": "*",
```

Comment:
Best to put actual versions here, maybes prefer lowest much faster.
Also phpstan ^1.2.0 etc

Reply:
Fixed.